test/conformance/gateway_conformance_test.go

```diff
@@ -23,7 +22,7 @@ var (
 	showDebug     = true
 	shouldCleanup = true
 
-	manifestRepo                  = "https://raw.githubusercontent.com/kubernetes-sigs/gateway-api/master/"
+	manifestRepo                  = "https://raw.githubusercontent.com/kubernetes-sigs/gateway-api/v0.5.0/"
```

Comment:
Can we introduce a similar mechanism as used for kustomize URL https://github.com/Kong/kubernetes-ingress-controller/blob/b421c0534909aa183ee946738f0236b886d292a5/test/consts/gateway.go#L4 ? i.e. put this into a `consts` package and generate it via a make target.

Reply:
nope because it is not deployed by us: `conformanceTestsBaseManifests` is given to the conformance test suite and it's up to the conformance testing framework to deploy the manifests.

Reply:
Hmm 🤔 I'm not sure how that's relevant. All I wanted to suggest is to make the value not be hardcoded here but to be generated based on the actual [dependency's version in `go.mod`](https://github.com/Kong/kubernetes-ingress-controller/blob/67f29ad41a3cebd1cae35c09cb5e49299102d7dc/go.mod#L43)

Reply:
Oh ok, I didn't get your point. I wasn't aware that the file is generated automatically. I created another PR to implement your suggestion as I'd avoid making this PR even bigger with this change. PTAL https://github.com/Kong/kubernetes-ingress-controller/pull/2802/files